packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/styles/index.ts

```diff
@@ -38,7 +39,7 @@ export default function styles(props: NoteBodyEditorProps) {
 			},
 			leftExtraToolbarContainer: {
 				...extraToolbarContainer,
-				width: 80,
+				width: { leftExtraToolbarContainerWidth },
```

Comment:
```suggestion
				width: leftExtraToolbarContainerWidth,
```

Including the `{`s sets `width` to an object with content
```
{
    leftExtraToolbarContainerWidth: leftExtraToolbarContainerWidth,
}
```
Setting `width` directly to `leftExtraToolbarContainerWidth` may make more sense here.

Reply:
Actually, when we directly update leftExtraToolbarContainerWidth (width: leftExtraToolbarContainerWidth), it's only setting the value once and not updating the width when watchedNoteFiles changes. I've tried logging the value of leftExtraToolbarContainerWidth, which shows correctly. However, the problem is that the width of the container is not updating during changes in watchedNoteFiles. I've tried to figure it out, and I believe the issue is that the styles are initially loaded and not updating during changes in watchedNoteFiles
So in [packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx](https://github.com/laurent22/joplin/pull/10069/files/76fb4682a350b7ea2b6b2498b1d268ef8c4e1e0e#diff-910138bf94c934ff21769a3dab9a765e3e0841b4a08fb8c295de183d048ff94a) we can do this which will load styles when watchedNoteFiles changes
![image](https://github.com/laurent22/joplin/assets/97527096/3dfe522c-9e3d-462f-94f9-b35561aee8b0)

Reply:
> I've tried to figure it out, and I believe the issue is that the styles are initially loaded and not updating during changes in watchedNoteFiles

It should be possible to fix this by updating the dependency array (`cacheKey`) passed to `buildStyles`. I think making this change in TinyMCE.tsx would also be fine, but making the change in `styles/index.ts` should keep the styles in closer to the same place.

Reply:
Thank you for pointing that out.I wasn't aware of the (`cacheKey`) concept before, but after your suggestion, I made the necessary changes in the `styles/index.ts`.